styles/typography.css

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+html,
+body {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Add the font in tailwind config file and use it as a class on the body element instead of targeting the body element directly

Reply:
I think to achieve that I must add a custom document file ``pages/_document.js`` that allows me to augment the overall HTML document to add a "body" tag which will be different in our project for the way nextjs handle pages using ``pages/_app.js``, so I was not sure that I was allowed to do that or not.